File: Assignments/Solutions/Shreya Kundu/Assignment 4/1.cpp
/*
Question 1

Given two strings str1 and str2, write program to swap two strings

I/P:
str1="C++" , str2="Java"
O/P:
str1="Java" , str2="C++"
*/
#include<bits/stdc++.h>
using namespace std;
int main(){
	string s1="Helluu";
	string s2="Hi";
	string s=s1;
	s1=s2;
	s2=s;
	cout<<s1<<"\n";
	cout<<s2<<"\n";
	return 0;
}

File: Assignments/Solutions/Utkarsh Rastogi/assignment4/1.py
'''Question 1

Given two strings str1 and str2, write program to swap two strings

I/P:
str1="C++" , str2="Java"
O/P:
str1="Java" , str2="C++"'''

str1='c++'
str2='java'
print(f'The original values are - str1={str1},str2={str2}')
temp=str1
str1=str2
str2=temp
print(f'After swapping the values are - str1={str1},str2={str2}')

#output
'''The original values are - str1=c++,str2=java
After swapping the values are - str1=java,str2=c++'''
